content/blog/posts/2022-07-01-accessibility-basics.md

```diff
@@ -0,0 +1,153 @@
+
+Someone without those same abilities could have a fairly similar experience if the page is built with their experience in mind. Based on our abilities as able-bodied developers, our instinct may be to add elements as we see them on a design spec.
+
+When building with accessibility in mind, a good rule of thumb is to start by thinking of what elements should be on the page _without_ any of the design in place. Ccreate the scaffolding for the content first, then add in the creative touches that differentiate your page.
+
+## Basic Considerations for A11y
+
+Here are some basics to pay attention to so you can jump-start accessibility for your web application:
+
+### Order Elements in Each Section Properly:
+
+![Section of shipshape.io with an image taking up the left half of the section and a header and two paragraphs making up the right half](/img/blog/image-before-header-section.png)
+
+Let's take a closer look at this section of our page. Because most languages read left to right, we start our attention on the left side of the page, which this time, is an image and might start coding like so:
+
+```html
+<img />
+<h1>Title</h1>
+<p>some body copy</p>
+```
+
+A header should be the leading element in its section; otherwise it can create confusion for non-GUI users. On shipshape.io, we've used [tailwindCSS](https://tailwindcss.com/docs/order) to help us style something like this easily, using the `order` utility. `order-last` forces the `div` with the header and text to come after the image, like so:
+
+```html
+<section class="flex">
+  <div class="order-last">
+    <h1>Title</h1>
+    <p>some body copy</p>
+  </div>
+  <div>
+    <img />
+  </div>
+</section>
+```
+
+### Use Proper H-Tags
+
+Accessibility tools rely on well-organized DOM and content to direct their users to where they want to get on a page. Screen reader users are given the option to listen to a list of headings on a page so they can skip around and get to their desired section for further interaction _without_ going through every single element on the page, line by line.
+
+An anti-pattern for using `h-tag`'s is to set each header level to a certain set of styles and sizes and using them for different text styles through the page. This promotes [using headers out of order](https://www.w3.org/WAI/tutorials/page-structure/headings/). Instead, first make sure your content is organized in proper order, then create the CSS styles to meet your design needs.
+
+![Section of shipshape.io with blocks of text using 5 different styles: large-font blue header, large-font white header, small, bolded white text within a button, small regular-weight blue body text, and small regular-weight white body text](/img/blog/h-tag-don't.png)
```

Comment:
```suggestion
<img src="/img/blog/h-tag-don't.png" align="left" style="padding: 2rem"></img>
```

Reply:
Hope this is closer to what you described but feel free to play around with it!

Reply:
Thank you, yes!